### PR TITLE
Normalization currency

### PR DESCRIPTION
**Pull request by hairtail**
## Summary
1. Replace `BigInt` & `toString` with `CurrencyUtils.decode` & `CurrencyUtils.encode` 
2. Use `bigint` for fee in `verifyCanSend`
3. Remove unnecessary convert.
## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish-cli/src/commands/deposit.ts [hunk 1]
```diff
@@ -119,10 +119,14 @@ export default class Bank extends IronfishCommand {
...
 
     const balanceResp = await this.client.getAccountBalance({ account: accountName })
-    const confirmedBalance = BigInt(balanceResp.content.confirmed)
-    const requiredBalance = CurrencyUtils.decodeIron(IRON_TO_SEND) + BigInt(fee)
+    const confirmedBalance = CurrencyUtils.decode(balanceResp.content.confirmed)
+    const requiredBalance = CurrencyUtils.decodeIron(IRON_TO_SEND) + fee
     if (confirmedBalance < requiredBalance) {
-      this.log(`Insufficient balance: ${confirmedBalance}. Required: ${requiredBalance}`)
+      this.log(
+        `Insufficient balance: ${CurrencyUtils.encode(
```
Comment (NullSoldier) [inline ironfish-cli/src/commands/deposit.ts:126]: I think were thinking about a way to make this more clear, but you should use `CurrencyUtils.renderIron()` when displaying to humans.

File: ironfish-cli/src/commands/deposit.ts [hunk 2]
```diff
@@ -119,10 +119,14 @@ export default class Bank extends IronfishCommand {
...
+          confirmedBalance,
+        )}. Required: ${CurrencyUtils.encode(requiredBalance)}`,
```
Comment (NullSoldier) [inline ironfish-cli/src/commands/deposit.ts:128]: Same here with `CurrencyUtils.renderIron()`

Comment (hairtail) [reply ironfish-cli/src/commands/deposit.ts]: fixed

**Comment by jowparks** [comment]
Thanks for pulling in the changes from staging. LGTM

**Review by jowparks — APPROVED** [review]

**Review by NullSoldier — APPROVED** [review]

**Pull request merged**